tensorrt_llm/llmapi/mpi_session.py

```diff
@@ -272,19 +275,51 @@ def __init__(self, addr: str):
         self.queue = ZeroMqQueue(addr, is_server=False)
         self._is_shutdown = False
 
-    def submit(self, task: Callable[..., T], *args, **kwargs) -> list:
+    def submit(self,
+               task: Callable[..., T],
+               sync: bool = False,
+               *args,
+               **kwargs) -> None:
```

Comment:
Should still be the following?
```suggestion
               **kwargs) -> []:
```
Or you want to return `None`?

Reply:
Updated, thanks.